vcpkg/github-integration.md

```diff
@@ -0,0 +1,94 @@
+---
+title: GitHub Integration
+description: Information on the available vcpkg integration points with GitHub
+author: michaelbprice
+ms.author: miprice
+ms.date: 06/15/2023
+---
+
+# Available GitHub Integrations
+
+## Caching vcpkg-built binaries for your GitHub Actions workflows
+
+Repositories that use GitHub Actions for continuous integration can take advantage of [vcpkg's binary caching feature](./users/binarycaching.md) to reduce the amount of time it takes to build their projects. There are two binary cache providers that are available to GitHub repositories.
```

Comment:
What are the two binary cache providers? You might call them out rather than rely on the reader to take in the rest of the article to figure it out.

Reply:
Explicitly listing them out here should be in the next update.